simplq/src/api/requestFactory/index.js

```diff
@@ -15,10 +15,13 @@
  * useRequest  hook. (See src/api/useRequest.js for usage example)
  *
  */
-import * as QueueRequestFactory from './queue';
-import * as TokenRequestFactory from './token';
 
-export { QueueRequestFactory, TokenRequestFactory };
-
-export { getUserQueues, deleteQueue, getQueueStatus, getQueueStatusByName } from './queue';
```

Comment:
Just removed these cause I thought they've become redundant now.

Reply:
All these exports just to get `{ method: 'post', url: '/queue', data: { queueName } }`... It would much simpler to put it in corresponding hook for async action.

I'm considering ditching everything at the end of the day, but for now, this is good.